crates/bevy_winit/src/state.rs

```diff
@@ -505,7 +507,9 @@ impl<T: Event> ApplicationHandler<T> for WinitAppRunnerState<T> {
 
         if should_update {
             // Not redrawing, but the timeout elapsed.
-            if !self.ran_update_since_last_redraw {
+            // NOTE: Additional condition for WindowsOS.
```

Comment:
Are we certain that this is isolated to Windows?

Reply:
I've asked folk on discord to help test this on Linux and MacOS, I'll test it on WASM